packages/core/src/account/smartContractAccount.ts

```diff
@@ -115,11 +114,12 @@ export type ToSmartContractAccountParams<
   Name extends string = string,
   TTransport extends Transport = Transport,
   TChain extends Chain = Chain,
-  TEntryPointVersion extends EntryPointVersion = EntryPointVersion
+  TEntryPointVersion extends EntryPointVersion = DefaultEntryPointVersion
 > = {
   source: Name;
   transport: TTransport;
   chain: TChain;
+  entryPoint: EntryPointDef<TEntryPointVersion, TChain>;
```

Comment:
yea with this like this then setting the default for `TEntryPointVersion` `EntryPointVersion` works now